public/v4/apps/pocketbase.yml

```diff
@@ -0,0 +1,41 @@
+captainVersion: 4
+
+services:
+  "$$cap_appname":
+    caproverExtra:
+      dockerfileLines:
+        - FROM alpine:latest
```

Comment:
Suggest fixing this to at least a major version like `alpine:3`

Reply:
You are right!!!
Thanks, fix commit follow